packages/shared/wallet/src/settings-side-drawer/network-selector/network-selector.tsx

```diff
@@ -33,23 +32,29 @@ export const NetworkSelectorButton = () => {
         justifyContent: 'center',
       }}
     >
-      <TokenIcon symbol="eth" size="medium" />
-      <Box sx={{ marginLeft: theme.spacing(1) }}>
-        {chain?.id && chainEntities[chain.id] && chainEntities[chain.id].label}
-      </Box>
+      <TokenIcon symbol={'arb'} size="medium" />
+      <Box sx={{ marginLeft: theme.spacing(1) }}>{chains[0].label}</Box>
     </Box>
   );
 };
 
 export function NetworkSelector() {
```

Comment:
Is there a way that we don't have to duplicate all of this logic in two different files?